dcc-portal-ui/app/index.template.html

```diff
@@ -189,9 +190,9 @@ <h1 style="font-size: 4rem">
 <script src="bower_components/d3/d3.min.js"></script>
 <script src="bower_components/Blob/Blob.js"></script>
 <script src="bower_components/FileSaver/FileSaver.min.js"></script>
-<script src="bower_components/showdown/src/showdown.js"></script>
-<script src="bower_components/showdown/src/extensions/table.js"></script>
-<script src="bower_components/x2js/xml2json.js"></script>
+<script src="bower_components/showdown/<%= grunt.config('env') === 'production' ? 'compressed/Showdown.min.js' : 'src/showdown.js' %>"></script>
+<script src="bower_components/showdown/<%= grunt.config('env') === 'production' ? 'compressed/extensions/table.min.js' : 'src/extensions/table.js' %>"></script>
```

Comment:
Ah, this is the grunt task - https://www.npmjs.com/package/grunt-usemin

Unfortunately neither bower nor npm requires that level of metadata, e.g. here's angular's bower.json
https://github.com/angular/bower-angular/blob/master/bower.json
Notice that it only has a `main` that points to the unminified file

We _could_ write something ourselves, either by 
- option 1) creating a custom user-defined step that's like a middleware, when given a `*.js` file, it will try to find `*.min.js`, we could additionally provide it with configs with lookup tables for and explicitly give a path to the minified files of libraries that don't follow the conventional structure
- option 2) fork the library and hardcode our usecase to the existing [concat](https://github.com/yeoman/grunt-usemin/blob/master/lib/config/concat.js#L30-L32) task. (The library's deprecated with few updates anyway). 

Option 2 has fewer unknowns and removes the intermediary step (which involves file io since grunt doesn't do streams)